pylib/anki/consts.py

```diff
@@ -21,6 +21,9 @@
 QUEUE_TYPE_NEW = 0
 QUEUE_TYPE_LRN = 1
 QUEUE_TYPE_REV = 2
+QUEUE_TYPE_DAY_LRN = 3
+CARD_TYPE_RELEARNING = 3
+QUEUE_TYPE_DAY_LEARN_RELEARN = 3
```

Comment:
Could you merge the two duplicate QUEUE_TYPEs?

Reply:
Done. Even if I must state I'm surprised. I thought that the queue 3 means two distinct things in sched v1 and v2; so two names would have make sens